src/TiledArray/dist_eval/contraction_eval.h

```diff
@@ -104,6 +107,46 @@ namespace TiledArray {
 
     private:
 
+      // Static variable initialization ----------------------------------------
+
+
+      /// Initialize max_memory_ limit for SUMMA
+      static size_type init_max_memory() {
+        const char* max_memory = getenv("TA__SUMMA__MAX_MEMORY");
+        if(max_memory) {
+            // Convert the string into bytes
+            std::stringstream ss(max_memory);
+            double memory = 0.0;
+            if(ss >> memory) {
+                if(memory > 0.0) {
+                    std::string unit;
+                    if(ss >> unit) { // Failure == assume bytes
+                        if(unit == "KB" || unit == "kB") {
+                            memory *= 1024.0;
```

Comment:
let's use SI system(https://en.wikipedia.org/wiki/Binary_prefix): change 1024 -> 1e3, etc. Feel free to add "KiB", "MiB", "GiB" units